Extensions/EdFi.Ods.Profiles.Sample/EdFi.Ods.Profiles.Sample.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+<metadata>
+<id>EdFi.Ods.Profiles.Sample</id>
+<version>0.0.0</version>
+<authors>Ed-Fi Alliance</authors>
+<owners>Ed-Fi Alliance</owners>
+<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions</projectUrl>
+<requireLicenseAcceptance>false</requireLicenseAcceptance>
+<description>EdFi.Ods.Profiles.Sample</description>
+<releaseNotes></releaseNotes>
+<license type="file">LICENSE.txt</license>
+</metadata>
+<files>
+  <file src="../../../LICENSE.txt" />
+  <file src="Artifacts/**/*.sql" target="."/>
+  <file src="Artifacts/**/*.json" target="."/>
+  <file src="assemblyMetadata.json" target="."/>
```

Comment:
_assemblyMetadata.json_ should not be included in the nuspec file -- it should be an embedded resource in the assembly.